examples/server/server.cpp

```diff
@@ -4321,7 +4321,7 @@ int main(int argc, char ** argv) {
             });
         }
         res_ok(res, result);
-        res.status = 200; // HTTP OK
+        res.status = httplib::StatusCode::OK_200; // HTTP OK
```

Comment:
I think we can remove this, since `res_ok` already set the status code

Reply:
Ah yes, I'll update shortly. Thanks